android_client/ChitChat/app/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'jacoco-android'
+apply plugin: 'jacoco'
```

Comment:
I think this needs to be changed back to `jacoco-android` in order to fix the build. Could you do that quickly when you get a chance? @aakyol